shell/app/modules/cmp/common/alarm-strategy/index.tsx

```diff
@@ -133,21 +130,22 @@ export default ({ scopeType, scopeId, commonPayload }: IProps) => {
     // ]),
     {
       title: i18n.t('default:notification target'),
-      dataIndex: ['notifies', '0', 'notifyGroup'],
+      dataIndex: 'notifies',
       width: 400,
       className: 'notify-info',
       ellipsis: true,
-      render: (notifyGroup: COMMON_STRATEGY_NOTIFY.INotifyGroup) => {
+      render: (notifies: COMMON_STRATEGY_NOTIFY.INotifyGroupNotify[]) => {
         const tips = i18n.t('cmp:Notification group does not exist or has been remove. Please change one.');
+        if (notifies?.length > 0 && notifies[0]?.notifyGroup?.name) {
```

Comment:
`notifies?.length > 0 && notifies[0]?.notifyGroup?.name` to
`notifies?.length && notifies[0].notifyGroup?.name`